src/node/internal/validators.ts

```diff
@@ -394,6 +394,64 @@ export const kValidateObjectAllowObjectsAndNull =
   kValidateObjectAllowArray |
   kValidateObjectAllowFunction;
 
+/*
+  The rules for the Link header field are described here:
+  https://www.rfc-editor.org/rfc/rfc8288.html#section-3
+
+  This regex validates any string surrounded by angle brackets
+  (not necessarily a valid URI reference) followed by zero or more
+  link-params separated by semicolons.
+*/
+const linkValueRegExp = /^(?:<[^>]*>)(?:\s*;\s*[^;"\s]+(?:=(")?[^;"\s]*\1)?)*$/;
+
+export function validateLinkHeaderFormat(
```

Comment:
Honestly, I'd suggest just moving these directly into the file they are imported into. They are only ever likely to be used there.